### PR TITLE
Cleanup `SamlMessage` class

### PR DESCRIPTION
**Pull request by phlipper**
This includes some cleanups from working on other PRs. The commit messages should be self-explanatory.

### CODE REVIEW

File: lib/onelogin/ruby-saml/saml_message.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
-require 'cgi'
-require 'zlib'
-require 'base64'
+require "cgi"
```
Comment (Lordnibbler) [inline lib/onelogin/ruby-saml/saml_message.rb]: i'd opt for the opposite here, based on hound/rubocop -- double quote `"` only when using string interpolation, and single quote `'` everywhere else

**Comment by Lordnibbler** [comment]
nice aside from my comment -- can you adjust the quotes? sorry to nitpick

**Comment by phlipper** [comment]
@Lordnibbler no worries, I have updated the quotes.

**Comment by Lordnibbler** [comment]
:+1:

**Pull request merged**